### PR TITLE
fix(coding):  update instructions to CSS Flexbox challenge in Portuguese

### PR DESCRIPTION
**Pull request by brenoxav**
Change the positions of some words to be in accordance with the syntax rules of the Portuguese language.
Change some translated words back to English for clarity of the instructions.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/portuguese/01-responsive-web-design/css-flexbox/use-the-flex-direction-property-to-make-a-row.portuguese.md [hunk 1]
```diff
@@ -7,17 +7,19 @@ localeTitle: Use a propriedade flex-direction para criar uma linha
...
+<section id="description"> Adicionando <code>display: flex</code> a um elemento o transforma em um contêiner flexível. Isso torna possível alinhar quaisquer filhos desse elemento em linhas ou colunas. Você faz isso adicionando a propriedade <code>flex-direction</code> ao item pai e configurando-a para <code>row</code> ou <code>column</code>. A criação de uma linha alinhará os filhos horizontalmente e a criação de uma coluna alinhará os filhos verticalmente.
+Outras opções para <code>flex-direction</code> são  <code>row-reverse</code> (linha reversa) e <code>column-reverse</code> (coluna reversa).
```
Comment (thefrontendwizard) [inline curriculum/challenges/portuguese/01-responsive-web-design/css-flexbox/use-the-flex-direction-property-to-make-a-row.portuguese.md]: Please try to follow the same formatting of the previous translation. Don't break the line when there's no line break previously.

**Review by frontendwizard — CHANGES_REQUESTED** [review]

File: curriculum/challenges/portuguese/01-responsive-web-design/css-flexbox/use-the-flex-direction-property-to-make-a-row.portuguese.md [hunk 2]
```diff
@@ -7,17 +7,19 @@ localeTitle: Use a propriedade flex-direction para criar uma linha
...
+<strong>Nota</strong> <br> O valor padrão da propriedade <code>flex-direction</code> é <code>row</code>. </section>
```
Comment (thefrontendwizard) [inline curriculum/challenges/portuguese/01-responsive-web-design/css-flexbox/use-the-flex-direction-property-to-make-a-row.portuguese.md]: Remove line break.

**Review by frontendwizard — CHANGES_REQUESTED** [review]

**Comment by RandellDawson** [comment]
@thefrontendwizard At this point, I think we should either close this PR due to it being stale or just remove the added line breaks if that is the only thing wrong with this challenge.

**Comment by thefrontendwizard** [comment]
I'll remove the line breaks and merge it in.

**Comment by thefrontendwizard** [comment]
I don't have push access to the branch. I'll close this one.

**Pull request closed without merging**